public/components/agents/vuls/inventory/table.tsx

```diff
@@ -194,27 +163,26 @@ export class InventoryTable extends Component {
     };
 
     return (
-      <EuiFlexGroup>
-        <EuiFlexItem>
-          <EuiBasicTable
-            items={items}
-            error={error}
-            columns={columns}
-            pagination={pagination}
-            onChange={this.onTableChange}
-            rowProps={getRowProps}
-            sorting={sorting}
-            itemId="vulnerability"
-            isExpandable={true}
-            loading={isLoading}
-          />
-        </EuiFlexItem>
-      </EuiFlexGroup>
+        <TableWithSearchBarWzAPI
+          title='Vulnerabilities'
+          tableColumns={columns}
+          tableInitialSortingField='name'
+          searchTable={true}
+          searchBarSuggestions={this.suggestions}
+          endpoint={`/vulnerability/${this.props.agent.id}`}
+          reload={isLoading}
```

Comment:
This props in the original table, redo the request with the current table status, when `reload` prop change.